### PR TITLE
OOP school library: add basic UI

### PR DESCRIPTION
**Pull request by Laylo309**
- Create a `main.rb` file that will serve as your console app entry-point. It should have methods that do the following:
  - List all books.
  - List all people.
  - Create a person (teacher or student, not a plain `Person`).
  - Create a book.
  - Create a rental.
  - List all rentals for a given person id.
- In your `main.rb` define the entry point, this will be a method called `main` that is invoked at the end of your file. This method should do the following:
  - Present the user with a list of options to perform.
  - Lets users choose an option.
  - If needed, ask for parameters for the option.
  - Have a way to quit the app.

### CODE REVIEW

File: main.rb [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+
+    author = gets.chomp
+
+    @books << Book.new(title, author)
+    puts 'Book created'
+  end
+
+  def create_rental
+    print 'Select a book from the following list by number'
+    @book.each_with_index do |book, index|
```
Comment (henatan99) [inline main.rb]: - [x] The instance variable here should be `@books` not `@book`, as a result, an error is thrown when option 5 is selected in the user interface. Kindly fix this. :pray: 

Screenshot from 2021-11-20 00-09-29

File: main.rb [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+    name = gets.chomp
+
+    person =
+      case person_type
+      when '1'
+        print 'Has parent permission? [Y/N]: '
+        parent_permission = gets.chomp
+        parent_permission = parent_permission.downcase == 'y'
+
+        Student.new(age, name, parent_permission)
```
Comment (henatan99) [inline main.rb:49]: -  [x] When I try to create a student it throws an argument error. Kindly fix this issue. :pray: 

Screenshot from 2021-11-20 00-04-53

**Review by henatan99 — CHANGES_REQUESTED** [review]
Hi @Laylo309 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: main.rb (hunk 1 above)
Comment (Laylo309) [reply main.rb]: Done ✔️

File: main.rb (hunk 2 above)
Comment (Laylo309) [reply main.rb:49]: Done ✔️

File: main.rb (hunk 1 above)
Comment (henatan99) [reply main.rb]: Well done! :clap: :clap: :clap:

File: main.rb (hunk 2 above)
Comment (henatan99) [reply main.rb:49]: Great job! :+1: :+1: :+1:

**Review by henatan99 — APPROVED** [review]
Hi @Laylo309 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**